bin/spice/cmd/refresh.go

```diff
@@ -17,17 +17,46 @@ limitations under the License.
 package cmd
 
 import (
+	"encoding/json"
 	"fmt"
 
 	"github.com/spf13/cobra"
 	"github.com/spiceai/spiceai/bin/spice/pkg/api"
 	"github.com/spiceai/spiceai/bin/spice/pkg/context"
+	"github.com/spiceai/spiceai/bin/spice/pkg/spec"
+)
+
+const (
+	refreshSqlFlag  = "refresh-sql"
+	refreshModeFlag = "refresh-mode"
 )
 
 type DatasetRefreshApiResponse struct {
 	Message string `json:"message,omitempty"`
 }
 
+type DatasetRefreshApiRequest struct {
+	SQL  *string `json:"refresh_sql,omitempty"`
```

Comment:
```suggestion
	RefreshSQL  *string `json:"refresh_sql,omitempty"`
```

Let's be explicit in the code as well, SQL isn't very descriptive.

Reply:
Here and below